webpack.config.js

```diff
@@ -5,9 +5,11 @@ const OptimizeCssAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 const FileManagerPlugin = require("filemanager-webpack-plugin");
-const appTitle = "COSRI Patient Search";
+/* TODO - use PROJECT_NAME in appTitle */
```

Comment:
@achen2401 Can you please resolve these 2 TODOs?  I don't know how to pull in app config at this layer.